scripts/userstyles.yml

```diff
@@ -179,16 +182,15 @@ userstyles:
         > [!NOTE]
         > This theme applies to [AniChart](https://anichart.net/) as well, which is an extension of AniList.
     current-maintainers: [*anubisnekhet]
-  arch-wiki:
-    name: Arch Wiki
-    categories: [wiki, productivity]
+  arch-linux:
+    name: Arch Linux
+    categories: [system, wiki, development]
     icon: archlinux
     color: blue
     readme:
-      app-link: "https://wiki.archlinux.org/"
-      usage: "Make sure to use the default **Light** theme"
-    current-maintainers: []
-    past-maintainers: [*genshibe]
```

Comment:
You should move this past-maintainers line to the new combined arch-linux entry.